docs/devel/networking.md

```diff
@@ -379,5 +379,6 @@ VM instance. Despite that, the only IP address reported is the IPv6 address,
 which implicitly highly encourages IPv6 communication towards the VM instance.
 
 On a final note, there is a difference that impacts the user experience when
-using masquerade binding for IPv6 addresses; the VMI IP must be [manually
-configured by the user](https://kubevirt.io/user-guide/#/creation/interfaces-and-networks?id=masquerade-ipv6-support).
+using masquerade binding for IPv6 addresses: until we introduce support of IPv6
+[router advertisement](https://github.com/kubevirt/kubevirt/pull/4581),
```

Comment:
I wouldn't add a link to this unfinished PR (for some reason it is not a draft, but it should be). Anyway, RA along is not enough. We need stateful dhcp since we need to control the full address of the vm and need to send extra parameters like nameservers. So for completeness I would say -
[router advertisement](https://github.com/kubevirt/kubevirt/pull/4581) + [dhcpv6](https://github.com/kubevirt/kubevirt/pull/4555)

Reply:
I would not link them, and stick this PR's scope to fixing the URL. 

Not sure we want to track features (or provide means to do so) in the developer's guide.

Reply:
These PRs are up in the air, which means that the doc here is soon(?) to be out of date. It think it is important to give the reader a tool to check if the text is still valid.